### PR TITLE
Fixed some issues and added zh_cn.json

### LINKED ISSUE #2: Pale Oak Sapling cannot grow
I discovered that Pale Oak Sapling cannot grow.

According to the code about ConfiguredFeature definition, I checked the file directory data/iwie/worldgen/configured_feature/ and found no pale_tree.json file;

`public static final SaplingGenerator PALE;`
`public static final RegistryKey<ConfiguredFeature<?, ?>> PALE_REGISTRY = RegistryKey.of(RegistryKeys.CONFIGURED_FEATURE, Identifier.of("iwie", "pale_tree"));`
`static {PALE = new SaplingGenerator("pale_tree", Optional.of(PALE_REGISTRY), Optional.empty(), Optional.empty());}`

### PR DESCRIPTION
**Pull request by SAGUMEDREAM**
* Added zh_cn.json translation file
* Restricted Creaking Heart to spawn Creaking only at night
* Creaking spawned by Creaking Heart will die during the day
* Distinguish whether Creaking is spawned manually or by Creaking Heart
* Fixed the problem that Pale saplings cannot grow
* Added basic sounds for Creaking

### CODE REVIEW

**Comment by SAGUMEDREAM** [comment]
Another thing is that the Creaking created by the Creaking Heart will also die if it is 16 blocks away from the Creaking Heart.

File: src/main/java/rs/onako2/iwie/entity/CreakingEntity.java [hunk 1]
```diff
@@ -28,6 +41,37 @@ public static DefaultAttributeContainer.Builder createMobAttributes() {
                 .add(EntityAttributes.GENERIC_ATTACK_DAMAGE, 3.0);
     }
 
+    @Override
+    public void tick() {
+        super.tick();
+        if(this.getWorld().isClient()) {
+            return;
+        }
+        if(this.isHeartSpawn && this.getWorld().isDay() || (this.isHeartSpawn && !Util.isHeartNear(this, this.getWorld(), 16))) {
```
Comment (Onako2) [inline src/main/java/rs/onako2/iwie/entity/CreakingEntity.java]: Hi!
Seems like checking every tick if there is a creaking heart makes the server lagg.
grafik

Theoretically we could store in the nbt the coordinates and then check if that heart is still there and if not then check?

**Review by Onako2 — CHANGES_REQUESTED** [review]

File: src/main/java/rs/onako2/iwie/entity/CreakingEntity.java (hunk 1 above)
Comment (SAGUMEDREAM) [reply src/main/java/rs/onako2/iwie/entity/CreakingEntity.java]: Ok, let me try this solution.

Comment (SAGUMEDREAM) [reply src/main/java/rs/onako2/iwie/entity/CreakingEntity.java]: Now I changed the code according to the solution you provided, and I updated it to my fork repository, but now I don’t know how to pull the request again on this page

**Comment by SAGUMEDREAM** [comment]
I have synchronized the correct block distance between Creaking and the heart according to the wiki,
Can't use the NameTag to Creaking,
Illager will run away if it encounters Creaking within 8 blocks

**Review by Onako2 — APPROVED** [review]
Thanks for contributing! <3

**Pull request merged**